app/charts/area/chart-area.tsx

```diff
@@ -35,6 +27,8 @@ import {
 } from "@/graphql/query-hooks";
 import { useLocale } from "@/locales/use-locale";
 
+import { ChartLoadingWrapper } from "../scatterplot/chart-loading-wrapper";
+
```

Comment:
I think this wrapper shouldn't be located in scatterplot 😅

Reply:
Done :)